.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+This template is for bug reports. If you are here for another reason, please see below:
+
+1. To propose a new rule, see http://eslint.org/docs/developer-guide/contributing/new-rules
+2. To request a change, see http://eslint.org/docs/developer-guide/contributing/changes
```

Comment:
I suppose that's a danger, but I think it will be more confusing if those types of reports aren't included.  Maybe it will help reduce the "wall of text" if we make each section header an H2 (##) or at least H3 (###)?